apps/liquidity-provision-dashboard/src/app/app.tsx

```diff
@@ -1,18 +1,46 @@
+import type { InMemoryCacheConfig } from '@apollo/client';
+import { NetworkLoader, useInitializeEnv } from '@vegaprotocol/environment';
 import { useRoutes } from 'react-router-dom';
 
 import '../styles.scss';
 import { Navbar } from './components/navbar';
 
 import { routerConfig } from './routes/router-config';
 
+const cache: InMemoryCacheConfig = {
```

Comment:
**thought (non-blocking)**: @asiaznik added a shared config in https://github.com/vegaprotocol/frontend-monorepo/pull/2914, which might be useful. Or they might be totally separate. Worth a quick look anyway - perhaps there's some benefit to all apps ffrom these cache changes.

Reply:
This config looks different, to the new default one. I'll leave this as is for now. Don't wan't to introduce anything unexpected